features/admin.connections.v1/components/edit/settings/general-settings.tsx

```diff
@@ -33,11 +33,13 @@ import { Dispatch } from "redux";
 import { CheckboxProps, Divider, List } from "semantic-ui-react";
 import {
     deleteConnection,
+    deleteCustomAuthentication,
```

Comment:
Update the year range in the license header for every file that has been changed.